### PR TITLE
[zh] Sync #14434 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14434 into Chinese, keep the same format with context not translate the `volume`

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/customize-installation/index.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ spec:
 1. [Strategy](https://kubernetes.io/zh-cn/docs/concepts/workloads/controllers/deployment/)
 1. [Env](https://kubernetes.io/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container/)
 1. [Pod security context](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
+1. [Volumes and volume mounts](https://kubernetes.io/zh-cn/docs/concepts/storage/volumes/)
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/customize-installation/index.md:105]:
```suggestion
1. [卷和卷挂载](https://kubernetes.io/zh-cn/docs/concepts/storage/volumes/)
```
这个 list 的一些词条要不要翻译一下

Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/customize-installation/index.md:105]: I think so.

Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/customize-installation/index.md:105]: But just check this list, a lot of links need to be updated, I think just merge this no translation PR, I can create another PR for fix links and translate both EN and CN content.

Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/customize-installation/index.md:105]: ok, all right

**Review by windsonsea — APPROVED** [review]
thanks.

**Pull request merged**